docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+* any decisions by members of the ``sktime`` developer community, in their execution of an official ``sktime`` role.
+  This especially includes core developer, Community Council, and Code of Conduct Committee roles.
+
+Examples for in-scope:
+
+* an academic allocating grant monies from a grant, or a data science consultant providing consultancy services, with a primary ``sktime`` branding
+* hiring decisions for roles that are advertised with a primary ``sktime`` branding
+* spending decisions from ``sktime`` community administered accounts
+* decisions arising from academic collaboration or commercial requests directed to the ``sktime`` community,
+  e.g., via official email, or via digital ``sktime`` discussion tools (slack, GitHub discussion etc)
+
+"resources" in the above context are to be interpreted in the usual sense of the word,
+i.e., valuable possessions, opportunities, qualities, that can be accessed and allocated.
+
+Examples include but are not limited to:
+
+* budget, money, financial credit
+* job opportunities, business opportunities
+* decision making power, soft or hard influence
+* dedicated work time of subordinates or dependents
+* material possessions of value
+* information of value
+* access to any of the above
+
+Best practice
+^^^^^^^^^^^^^
+
+To ensure decision making in line with the code of conduct, decision making must be:
+
+* by the ``sktime`` community, through ``sktime`` community decision making mechanisms.
+  E.g., following decision making outlined in the ``sktime`` governance document.
+* in communication transparent to the ``sktime`` community, following communication requirements outlined in the ``sktime`` governance document.
+* pursuant of and in line with the charitable mission of ``sktime`` and NumFOCUS.
+
+Adherence to widely accepted guidelines of anti-corruption and anti-bribery practice
+(e.g., United Nations Anti-Corruption Guidelines) is strictly expected,
+especially for resource allocation decisions of major magnitude, e.g., in hiring processes,
+or decisions in the 5-digit dollar range and upwards.
+
+This strict requirement for best practice remains unchanged even if a local context may require less,
+e.g., university administrations,
+company policies, national anti-corruption and anti-bribery laws (or their absence), etc.
+
+For instance, it is in-principle possible to misappropriate ``sktime`` resource, while
+being in formal compliance with specific national laws, institutional laws, and local policies.
+Irrespective of this, such an action would still count as a severe CoC violation.
+
+Examples of not acceptable behaviour
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* obtaining a resource opportunity through ``sktime``,
+  then retreating or resigning ``sktime`` roles to declare that the opportunity is now in a personal role and not on behalf of ``sktime``.
+* pretending to communicate on behalf of ``sktime``, or unauthorized use of the ``sktime`` brand,
```

Comment:
Agree that this is an inappropriate behaviour. However, the 'unauthorised' again goes against the open-source nature of the project. We do not authorise anyone to use the package - everyone is free to do so. With the brand and the logo - we should have (already have?) the TM to 'authorise' its use, otherwise we're not a company and can't stop anybody from using sktime as a brand if I'm not mistaken.

Reply:
I almost agree with you @GuzalBulatova, but I think you are not entirely correct about two points.

Legal point (best of my knowledge):

> we should have the TM to 'authorise' its use

We do not need the TM, clause 3 of the BSD-3 license suffices, and we can say "authorized or not" (see above)

Moral point:

> However, the 'unauthorised' again goes against the open-source nature of the project.

I disagree with this, open source does not mean anarchy or laissez faire.

That's why we have a code of conduct.

We have the legal right to control who says they are or represent ``sktime``, and morally we should not just give that away to anyone under any condition. E.g., someone who ticks all the possible boxes in violating our code of conduct.

Reply:
shorter summary:

* you are right, there is no authorization on use of software package and this would indeed contradict open source
* what this all refers to is *authorization on use of brand* (not software package). See discussion above.